### ISSUE TITLE
Pod integration fail

### ISSUE THREAD

**Issue by robertodaviduk** [kishikawakatsumi/BitcoinKit#260]
Hi, i am getting the next error when execute "pod install"..

fatal error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo: /var/folders/xf/b7gq41hs2jv_4243r8t57l4m0000gn/T/tmp.mZ7MzT8O/.build/iphoneos/lib/libsecp256k1.a and /var/folders/xf/b7gq41hs2jv_4243r8t57l4m0000gn/T/tmp.mZ7MzT8O/.build/iphonesimulator/lib/libsecp256k1.a have the same architectures (i386) and can't be in the same fat output file

Found a possible solution adding this 2 lines to build_secp256k1.sh file but it doesn't work...

// lines 23 and 24.
(cd src && make distclean > /dev/null) 
(cd src && ./autogen.sh)

How can I fix it?

Thanks

**Comment by Marlunes** [comment]
Hey man, got solution on this?

**Comment by SurpriseMF3000** [comment]
Same Issue here.
Do you solved it?

**Comment by robertodaviduk** [comment]
No, i didnt try again, sorry.. Maybe SPM works better than cocoapods ( in my case, I prefere SPM )
